### ISSUE TITLE
Saved model name not customizable 

### ISSUE THREAD

**Issue by ke-lara** [Shivanandroy/simpleT5#37]
`    def training_epoch_end(self, training_step_outputs):
        """ save tokenizer and model on epoch end """
        self.average_training_loss = np.round(
            torch.mean(torch.stack([x["loss"] for x in training_step_outputs])).item(),
            4,
        )
        path = f"{self.outputdir}/simplet5-epoch-{self.current_epoch}-train-loss-{str(self.average_training_loss)}-val-loss-{str(self.average_validation_loss)}"`

Will be very helpful if you can allow the name customizable (note the 'path' assignment). 

Btw, SimpleT5 is simply cool!
